apps/server/src/modules/task/uc/task.uc.ts

```diff
@@ -225,4 +230,84 @@ export class TaskUC {
 		await this.taskRepo.delete(task);
 		return true;
 	}
+
+	async find(userId: EntityId, taskId: EntityId) {
+		this.checkIndividualTaskEnabled();
+
+		const [user, task] = await Promise.all([
```

Comment:
Shouldn't be the promises awaited one after the other? The authorization is a 	precondition for the repo find operation.

Reply:
you are right, it makes sense to do them one after another. Was like this in other methods already, but I will fix those too.